xsd-parser-rs/src/tests/type_name_clash/input.xsd

```diff
@@ -4,7 +4,7 @@
            targetNamespace="http://example.com"
            elementFormDefault="qualified">
     <xs:complexType name="BarType">
-        <xs:attribute name="a" type="xs:string" />
+        <xs:attribute name="Field" type="xs:string" />
```

Comment:
Better leave `a` for consistency because there is still another attribute called `b`